From f9556dce8a046ba0aa6a9665186e6a2c26ee0d33 Mon Sep 17 00:00:00 2001
From: Mikhail Konovalov <mkonovalov@elvees.com>
Date: Mon, 15 Jan 2024 18:13:42 +0300
Subject: [PATCH] tools: Makefile: Enable RSASSA PSS

To properly support signed FIT images it is required to enable
rsassa-pss signature scheme.

Issue: #IPCAMSW-3917
Upstream: N/A (there are no working upstream patches)
Change-Id: Ia1ec3c13a5857848495f91610e0fe5b68645b7a3
Signed-off-by: Mikhail Konovalov <mkonovalov@elvees.com>
---
 tools/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/Makefile b/tools/Makefile
index d9942533c9..58383a4e7c 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -166,6 +166,7 @@ ifdef CONFIG_TOOLS_LIBCRYPTO
 HOST_EXTRACFLAGS	+= -DCONFIG_TOOLS_FIT_SIGNATURE
 HOST_EXTRACFLAGS	+= -DCONFIG_TOOLS_FIT_SIGNATURE_MAX_SIZE=0xffffffff
 HOST_EXTRACFLAGS	+= -DCONFIG_TOOLS_FIT_CIPHER
+HOST_EXTRACFLAGS	+= -DCONFIG_TOOLS_FIT_RSASSA_PSS
 endif
 
 # MXSImage needs LibSSL
-- 
2.39.1

